Makefrag

```diff
@@ -24,16 +27,47 @@ include $(testchip_dir)/Makefrag
 
 CHISEL_ARGS ?=
 
-FIRRTL_FILE=$(build_dir)/$(PROJECT).$(MODEL).$(CONFIG).fir
-ANNO_FILE=$(build_dir)/$(PROJECT).$(MODEL).$(CONFIG).anno.json
-VERILOG_FILE=$(build_dir)/$(PROJECT).$(MODEL).$(CONFIG).v
+long_name=$(PROJECT).$(MODEL).$(CONFIG)
+
+FIRRTL_FILE  ?=$(build_dir)/$(long_name).fir
+ANNO_FILE    ?=$(build_dir)/$(long_name).anno.json
+VERILOG_FILE ?=$(build_dir)/$(long_name).top.v
+HARNESS_FILE ?=$(build_dir)/$(long_name).harness.v
+SMEMS_FILE   ?=$(build_dir)/$(long_name).mems.v
+SMEMS_CONF   ?=$(build_dir)/$(long_name).mems.conf
+
+REPL_SEQ_MEM = --repl-seq-mem -c:$(MODEL):-o:$(SMEMS_CONF)
+
+# This should match whatever is in build.sbt
```

Comment:
What does this comment mean? I don't see a barstools version in the build.sbt. I guess its using the default version. Maybe mention that quickly in the comment.

Reply:
Yes, it's the commonSettings version. Updated the comment.